2.1/runtime/nanoserver-1709/amd64/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# escape=`
+
+# Installer image
+FROM microsoft/windowsservercore:1709 AS installer-env
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+# Retrieve .NET Core Runtime
+ENV DOTNET_VERSION 2.1.0-preview1-25816-02
+ENV DOTNET_DOWNLOAD_URL https://dotnetcli.blob.core.windows.net/dotnet/Runtime/$DOTNET_VERSION/dotnet-runtime-$DOTNET_VERSION-win-x64.zip
+ENV DOTNET_DOWNLOAD_SHA 1F7BB23909CBBEA1A010773FD9E7BFE9944E551FEB7DCF10AC5D923901BD13BEB09A2677D528F16398329FD892B4B2975404D842581475C703633C99DD069CC5
+
+RUN Invoke-WebRequest $Env:DOTNET_DOWNLOAD_URL -OutFile dotnet.zip; `
+    if ((Get-FileHash dotnet.zip -Algorithm sha512).Hash -ne $Env:DOTNET_DOWNLOAD_SHA) { `
+        Write-Host 'CHECKSUM VERIFICATION FAILED!'; `
+        exit 1; `
+    }; `
+    `
+    Expand-Archive dotnet.zip -DestinationPath dotnet; `
+    Remove-Item -Force dotnet.zip
+
+
+# Runtime image
+FROM microsoft/nanoserver:1709
+
+COPY --from=installer-env ["dotnet", "C:\\Program Files\\dotnet"]
+
+RUN setx PATH "%PATH%;C:\Program Files\dotnet"
```

Comment:
I think it is correct, but clarifying - should backslash in `C:\Program Files\dotnet` be escaped?

Reply:
Escaping is only necessary with the `[ ]`  syntax.  BTW the combination of our builds and utests will catch issues here.  If this syntax were incorrect we couldn't do things like `RUN dotnet new`